charts/kubeai/templates/_helpers.tpl

```diff
@@ -60,3 +60,14 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create the name of the huggingface secret to use
+*/}}
+{{- define "kubeai.huggingfaceSecretName" -}}
+{{- if .Values.secrets.huggingface.secretName -}}
+huggingface: {{ .Values.secrets.huggingface.secretName }}
```

Comment:
Its a little confusing to me to have huggingfaceSecretName be a key/val pair. Can we change this helper to return a string?